charts/seed-bootstrap/charts/monitoring/templates/config.yaml

```diff
@@ -101,3 +101,23 @@ data:
     - job_name: prometheus
       static_configs:
       - targets: [ localhost:9090 ]
+
+    - job_name: vpa-recommender
+      kubernetes_sd_configs:
+        - role: pod
+          relabel_configs:
+            - source_labels: [__meta_kubernetes_pod_label_app]
+              regex: ^vpa-recommender$
```

Comment:
regular expressions are anchored by default, so that you can drop the `^` and `$`

Reply:
done.